openwisp_monitoring/device/tests/test_check_signals.py

```diff
@@ -0,0 +1,48 @@
+from django.test import TestCase
```

Comment:
We have a file which contains some tests for `apps.py`, I think we can add a test there instead of creating a new test class just for this.